.github/workflows/size-limit.yml

```diff
@@ -17,9 +17,7 @@ jobs:
         with:
           node-version: 16
           cache: yarn
-      - run: yarn --frozen-lockfile
-      - run: yarn build
```

Comment:
This is already taken care of by the `size-limit-action` and in fact allows a custom `build_script` which I leverage to only build `@shopify/polaris` and its dependencies.